package.json

```diff
@@ -9,37 +9,19 @@
   "exports": {
     ".": {
       "import": "./dist/nodejs.mjs",
-      "require": "./dist/nodejs.js",
-      "types": "./dist/nodejs.d.ts",
-      "browser": "./dist/nodejs.mjs",
-      "bun": "./dist/nodejs.mjs",
-      "deno": "./dist/nodejs.mjs",
-      "edge-light": "./dist/nodejs.mjs",
-      "edge-routine": "./dist/nodejs.mjs",
-      "netlify": "./dist/nodejs.mjs",
-      "react-native": "./dist/nodejs.mjs",
-      "wintercg": "./dist/nodejs.mjs",
-      "worker": {
-        "import": "./dist/cloudflare.mjs",
-        "types": "./dist/cloudflare.d.ts"
-      },
-      "workerd": {
-        "import": "./dist/cloudflare.mjs",
-        "types": "./dist/cloudflare.d.ts"
-      }
```

Comment:
we added these to automatically import cloudflare clients in worker environemnts. Why do we delete them?

Reply:
It uses the first import it matches, so i think these were not used before anyways.
https://nodejs.org/api/packages.html#conditional-exports